tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.NetCore.cs

```diff
@@ -77,7 +77,7 @@ private static Assembly ResolveAssembly(string name)
 
             // WARNING: Logs must not be added _before_ we check for the above bail-out conditions
             StartupLogger.Debug("Assembly Resolve event received for: {0}", name);
-            var path = Path.Combine(ManagedProfilerDirectory, $"{assemblyName.Name}.dll");
+            var path = new FileInfo(Path.Combine(ManagedProfilerDirectory, $"{assemblyName.Name}.dll")).FullName;
```

Comment:
As Andrew isn't here :-) `Path.GetFullPath` would be fewer allocations

Reply:
```suggestion
            var path = Path.GetFullPath(Path.Combine(ManagedProfilerDirectory, $"{assemblyName.Name}.dll"));
```